docs/setup/aws.md

```diff
@@ -93,19 +93,21 @@ collection of these environment variables:
 
 ```bash
 # example values
-export GRAPL_DEPLOYMENT_NAME="some-grapl-deployment-name"
+export GRAPL_DEPLOYMENT_NAME="grapl-deployment-name"
+export DEPLOYMENT_NAME="grapl-deployment-name"
```

Comment:
I'd prefer to normalize these in code, if possible, rather than having two different variable names for the same thing.

Reply:
Can we keep this for now and I'll open a new ticket to fix this, I don't want others to run into issues with deployments.

Reply:
Could we reference that GitHub issue from the docs, and just say the duplication is a workaround for now?

Reply:
https://github.com/grapl-security/grapl/pull/717